modules/io_bazel_rules_closure/0.15.0/MODULE.bazel

```diff
@@ -0,0 +1,164 @@
+module(
+    name = "io_bazel_rules_closure",
```

Comment:
Let's just call it rules_closure to keep it simple and consistent with other rules.